specs/_features/maxeb_increase/capella.py

```diff
@@ -1696,7 +1699,7 @@ def process_block_header(state: BeaconState, block: BeaconBlock) -> None:
 
     # Verify proposer is not slashed
     proposer = state.validators[block.proposer_index]
-    assert not proposer.slashed
+    assert not is_slashed_proposer(proposer)
```

Comment:
Should a slashed_attester be allowed to propose?

Reply:
IMO, it shouldn't

Reply:
```suggestion
    assert not is_slashed_proposer(proposer) and not is_slashed_attester(proposer)
```

Alternatively:
```python
assert not is_slashed(proposer)

...

def is_slashed(validator: Validator) -> bool:
    return is_slashed_proposer(validator) or is_slashed_attester(validator)
```